src/index.js

```diff
@@ -15,6 +15,25 @@ const validateFunctionOrUndefined = (param, paramName) => {
   if (!(typeof param === 'undefined' || typeof param === 'function')) throw new Error(`Please provide correct value for ${paramName}`)
 }
 
+const handleBulkResponseErrors = (bulkResponse, body) => {
+  if (bulkResponse.errors) {
+    const erroredDocuments = []
+    bulkResponse.items.forEach((action, i) => {
+      const operation = Object.keys(action)[0]
+      if (action[operation].error) {
+        erroredDocuments.push({
+          status: action[operation].status,
+          error: action[operation].error,
+          operation: body[i * 2],
+          document: body[i * 2 + 1]
+        })
+      }
+    })
+    console.error(erroredDocuments)
```

Comment:
I guess it shouldn't be here. If we would like to log something, it would be good to add configuration for it. Seems to be more comprehensive, so probably for now console.log should be removed.

Reply:
I might try to see if I set DEBUG=elasticsearch when running the tests I get meaningful information on the issues with the batch, otherwise, those are really cryptic messages.